src/AbpHelper.Core/Commands/Generate/Crud/CrudCommand.cs

```diff
@@ -107,7 +107,7 @@ protected override IActivityBuilder ConfigureBuild(CrudCommandOption option, IAc
 
                                     @switch.When(UiFramework.Angular.ToString("D"))
                                         // TODO
-                                        //.AddUiAngularGenerationWorkflow()
+                                        .AddUiAngularGenerationWorkflow()
```

Comment:
I think this change should roll back since it is not related to the current PR. Maybe we can discuss the `AddUiAngularGenerationWorkflow()` later.

Reply:
thanks, you right